From 3f2d3d95bf73432a112b51e9cb88c21660468d57 Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Sat, 9 Apr 2022 02:44:46 +0000
Subject: [PATCH] tests: Disable blkid cache usage

Using the blkid cache can cause issues when running many file system tests
in parallel. We do not need it, as its only there to improve performance,
and using the cache does not provide significant performance improvements.

Reference:https://git.savannah.gnu.org/cgit/grub.git/commit?id=e883cc6a282891783902daf2d564a5b5c7a6a5aa
Conflict:NA

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>

---
 tests/util/grub-fs-tester.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/util/grub-fs-tester.in b/tests/util/grub-fs-tester.in
index bc14a05..fb5022f 100644
--- a/tests/util/grub-fs-tester.in
+++ b/tests/util/grub-fs-tester.in
@@ -1,6 +1,7 @@
 #!@BUILD_SHEBANG@
 
 set -e
+export BLKID_FILE=/dev/null
 
 fs="$1"
 
-- 
2.27.0

